functions/src/Services/Event.ts

```diff
@@ -1,12 +1,15 @@
-// const { App } = require('../app.js');
-// import * as App from 'firebase-admin';
 import { User } from "../Models/User";
 import { Event, eventConverter } from "../Models/Event";
 import { firestore } from 'firebase-admin';
 
-
+// Creates an event document and adds it to the events collection.
 export const createEvent = (event: any) => {
-	firestore().collection('events').withConverter(eventConverter).add(new Event(event));
+    // Retrieve the events collection with a converter to be able to convert generic types of data to Firestore data.
+    // Here, we convert event objects to Firestore data.
+    const eventCollection = firestore().collection('events').withConverter(eventConverter);
+
+    // Add new event document to events collection.
```

Comment:
I don't think this comment is necessary, The next line of code is self explanatory